### PR TITLE
fix duplicate SQL parsing and add docs

### PR DESCRIPTION
**Pull request by CalebDW**
**Changes**
The SQL Parser calls `Parser::parse` in the constructor so I removed the duplicate call.
https://github.com/phpmyadmin/sql-parser/blob/548382794d7571d42a5e3eeb81c4278aec8e50f9/src/Parser.php#L381

I also added some docs about Laravel 9 attributes and PostgreSQL dumps.

**Breaking changes**
N/A

### CODE REVIEW

**Comment by canvural** [comment]
Hi,

Thank you for this!

But maybe having a separate new features documentation in `docs/` folder would be better. To not clutter the README. Then we can reference it from README. What do you think?

**Comment by CalebDW** [comment]
@canvural, something I noted in the docs is that the sql parser can take a very long time parsing Postgres dumps (over 7 minutes for a <1MB file... only to return no tables). This is going to become a problem because the dumps are parsed automatically. I could point `squashedMigrationsPath` to a non-existent/empty directory but it would be more useful to just have an option disabling the parsing.

**Comment by szepeviktor** [comment]
Great work has been done on master. Please consider rebasing.

**Comment by szepeviktor** [comment]
Your commits are made with a non-GitHub email address.
If you want you can use your email address registered here on GitHub.

**Comment by CalebDW** [comment]
> Your commits are made with a non-GitHub email address. If you want you can use your email address registered here on GitHub.

Ah good catch, how do I update the commits?

**Review by canvural — CHANGES_REQUESTED** [review]

File: docs/features.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Rules
+
+All features that are specific to Laravel applications 
+are listed here.
+
```
[the remark this answers is not in the document]
Comment (CalebDW) [reply docs/features.md:5]: Ah good catch.

File: README.md [hunk 2]
```diff
@@ -109,6 +109,11 @@ To get around this a baseline file can be generated. The baseline file will crea
 
 A list of configurable rules specific to Laravel can be found [here](docs/rules.md).
 
+
+## Features
+
+A list of features specific to Laravel can be found [here](docs/features.md).
```
[the remark this answers is not in the document]
Comment (CalebDW) [reply README.md:115]: Fixed.

**Comment by canvural** [comment]
Thank you!

**Pull request merged**